jobs/kola-openstack.Jenkinsfile

```diff
@@ -43,97 +41,105 @@ properties([
     durabilityHint('PERFORMANCE_OPTIMIZED')
 ])
 
-currentBuild.description = "[${params.STREAM}][${params.ARCH}] - ${params.VERSION}"
+lock(resource: "kola-openstack-${params.ARCH}") {
 
-def s3_stream_dir = params.S3_STREAM_DIR
-if (s3_stream_dir == "") {
-    s3_stream_dir = "fcos-builds/prod/streams/${params.STREAM}"
-}
+    currentBuild.description = "[${params.STREAM}][${params.ARCH}] - ${params.VERSION}"
+    region = "ams1"
+    if (params.ARCH != "x86_64") {
+        region = "ca-ymq-1"
+    }
```

Comment:
Here it might be better to have a full if/elseif/else for each arch just so your brain doesn't have to thing too much when reading the code. Can still have a default region at the top (in case a new architecture ever gets thrown at us).